cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/ConsentWebView.java

```diff
@@ -97,6 +97,7 @@ private void setup() {
         }
         getSettings().setJavaScriptEnabled(true);
         this.setBackgroundColor(Color.TRANSPARENT);
+        this.requestFocus();
```

Comment:
why did we need this here?

Reply:
We need this because onBackPressed event will not be listen by web-view until user interacts with it or web-view have focus.